lib/plugin/css-variables-collector.js

```diff
@@ -24,6 +25,10 @@ CSSVariablesCollectorPlugin.prototype = {
 		return this.mixinStack.length > 0 || this.parenStack.length > 0;
 	},
 
+	_isInFontFaceDirective() {
+		return this.directiveStack.some(({name}) => name === "@font-face");
```

Comment:
I guess it would be faster to have a dedicated "font-face-directive" stack and check for it's length instead of searching the whole array of directives for every rule? Are directives a rare thing to have in the stack?

Reply:
Yeah, that makes sense.